main.py

```diff
@@ -61,5 +29,5 @@
     while 1:
         data = connection.recv(BUFFERSIZE)
         if not data: break
-        logging.INFO(data)
+        logging.info(data)
     connection.close()
```

Comment:
Might need to look at how Python handles error propagation. In Java world, it's possible that line isn't reached and we would use `try...catch...finally` to make sure the connection is closed. I would need to check Python rules - we can deal with this later.